extensions/arc/deployment/src/main/java/io/quarkus/arc/deployment/ArcConfig.java

```diff
@@ -69,6 +69,15 @@
     @ConfigItem(defaultValue = "true")
     public boolean transformUnproxyableClasses;
 
+    /**
+     * If set to true, if a private method that is neither an observer nor a producer and is annotated with an interceptor
```

Comment:
_"..., the build fails if a private..."_?

Reply:
Fixed